cpp/benchmarks/common/generate_input.cpp

```diff
@@ -65,45 +65,46 @@ T get_distribution_mean(distribution_params<T> const& dist)
 }
 
 // Utilities to determine the mean size of an element, given the data profile
-template <typename T>
-std::enable_if_t<cudf::is_fixed_width<T>(), size_t> avg_element_size(data_profile const& profile)
+template <typename T, std::enable_if_t<cudf::is_fixed_width<T>()>* = nullptr>
```

Comment:
Nitpick: Is `CUDF_ENABLE_IF()` available at this point?
```suggestion
template <typename T, CUDF_ENABLE_IF(cudf::is_fixed_width<T>())>
```